animesearch-ui/src/App.js

```diff
@@ -1,14 +1,49 @@
 import logo from './logo.svg';
 import './App.css';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
 
 function App() {
+  const [searchData, setSearchData] = useState()
+
+  // useEffect(() => {
```

Comment:
normally when you merge to master branch, the standard is to clean up this type of code (ie. if its not being used and commented out then delete it)